yt_dlp/extractor/manyvids.py

```diff
@@ -35,79 +52,41 @@ class ManyVidsIE(InfoExtractor):
             'ext': 'mp4',
             'title': 'MY FACE REVEAL',
             'description': 'md5:ec5901d41808b3746fed90face161612',
+            'thumbnail': 'https://ods.manyvids.com/1001061960/3aa5397f2a723ec4597e344df66ab845/screenshots/thumbs/custom_1_180_5be09c1dcce03.jpg',
             'uploader': 'Sarah Calanthe',
             'view_count': int,
             'like_count': int,
+            'release_date': '20181110',
+            'tags': ['EyeContact', 'Interviews', 'MaskFetish', 'MouthFetish', 'Redhead'],
+            'release_timestamp': 1541851200,
+            'duration': 224.0,
         },
     }]
 
     def _real_extract(self, url):
         video_id = self._match_id(url)
 
-        real_url = f'https://www.manyvids.com/video/{video_id}/gtm.js'
-        try:
-            webpage = self._download_webpage(real_url, video_id)
-        except Exception:
-            # probably useless fallback
-            webpage = self._download_webpage(url, video_id)
-
-        info = self._search_regex(
-            r'''(<div\b[^>]*\bid\s*=\s*(['"])pageMetaDetails\2[^>]*>)''',
-            webpage, 'meta details', default='')
-        info = extract_attributes(info)
+        info = traverse_obj(
+            self._download_json(f'https://www.manyvids.com/bff/store/video/{video_id}', video_id),
+            ('data', {dict})) or {}
```

Comment:
This `_download_json` call can/should be non-fatal, and let's do it after the `video_urls`/formats extraction so that we don't waste time before aborting if something is wrong